### ISSUE TITLE
forbidden properties can be edited if they are the only property of the block

### ISSUE THREAD

**Issue by 800020h** [JustS-js/debugStickSurvival#5]
**versions:**
- Minecraft 1.19.2
- Fabric Loader 0.14.21
- survival-debug-mod-1.0.1
- and a whole bunch of other mods too

with this config things like cake, and mangrove roots that only have one property can have that property modified even if it's forbidden

if the block has more than 1 property the forbidden properties can't be selected or modified

```json
{
  "allowed": {
    "blocks": [],
    "properties": [],
    "tags": []
  },
  "forbidden": {
    "blocks": [],
    "properties": [
      "waterlogged",
      "bites",
      "servings"
    ],
    "tags": [
      {
        "id": "minecraft:slabs"
      }
    ]
  },
  "messages": {
    "select": "Property (%s) was selected (%s).",
    "change": "Property (%s) was modified (%s).",
    "nomodify": "This block is not modifiable."
  },
  "whitelist": false
}
```
![2023-09-05_21 52 53](https://github.com/JustS-js/debugStickSurvival/assets/100995970/c052add0-44fc-4612-ba61-10e979e48480)
![2023-09-05_21 53 02](https://github.com/JustS-js/debugStickSurvival/assets/100995970/cf34b2fc-0bb2-45de-88fb-12c9203003b4)
I'm not sure why the characters before and after the property went like that after opening the config in vscode

**Comment by JustS-js** [comment]
The problem with characters might be connected with how i handle string formats, i look into that
As for the one property thing - yeah, that needs some fixing

**Comment by JustS-js** [comment]
Fixed in v1.2.0 for MC 1.21
